src/Application.vala

```diff
@@ -209,7 +239,8 @@ public class Terminal.Application : Gtk.Application {
     }
 
     private const OptionEntry[] entries = {
-        { "execute", 'e', 0, OptionArg.STRING_ARRAY, ref command_e, N_("Run a program in terminal"), "COMMAND" },
+        /* -e flag is used for running single string commands. May be more than one -e flag in cmdline */
+        { "execute", 'e', 0, OptionArg.STRING_ARRAY, ref command_e, N_("Run a program in terminal"), "PROGRAM_NAME" },
```

Comment:
You should add a description of `-x, --` here as well, right? :)

Reply:
Done

Reply:
I actually meant to add another entry for -x, that way the new argument will show up when you type `--help` :)

Reply:
Oh, I see. This causes some complications as you then have to support a long form option (I note that gnome-terminal does not document the -x or -- flag in the --help output).  However, I have implemented this now.